bricksrc/equipment.py

```diff
@@ -545,56 +548,19 @@
         # - is any specific kind of switch e.g. PoESwitch implied here?
         # Video_Wall (or should this be in a separate classification with displays and monitors?)
     },
-    "Safety": {
-        "tags": [TAG.Equipment, TAG.Security, TAG.Safety],
-        "subclasses": {
-            "Automated_External_Defibrillator": {
-                OWL.equivalentClass: BRICK["AED"],
-                "tags": [TAG.AED],
-                SKOS.definition: Literal(
-                    "Automated External Defibrillator. Used by trained people to help those experiencing cardiac issues."
-                ),
-            },
-            "AED": {
-                "tags": [
-                    TAG.Equipment,
-                    TAG.Security,
-                    TAG.Safety,
-                    TAG.AED,
-                    TAG.Defibrillator,
-                ],
-                SKOS.definition: Literal(
-                    "Automated External Defibrillator. Used by trained people to help those experiencing cardiac issues."
-                ),
-            },
-        },
-    },
-    "Intrusion": {
+    "Intrusion_Equipment": {
```

Comment:
I guess it should be something like `Intrusion_Detection_Equipment`? Can this be correct without `Detection` tag?

Reply:
I'm not quite sure that we'd want to add 'Detection' this high in the tree, since it's perfectly reasonable to call it 'Intrusion Control Equipment' or 'Intrusion management'. I'm not deadset against adding detection, just not convinced it's necessary.